oldScripts/gradescope_to_spreadsheet.py

```diff
@@ -331,4 +328,12 @@ def populate_instructor_dashboard():
     update_sheet_with_csv(output.getvalue(), sheet_api_instance, dashboard_sheet_id, 0, 3)
     output.close()
 
+def main():
+    if len(sys.argv) > 1:
+        creds = allow_user_to_authenticate_google_account()
+        gradescope_client = initialize_gs_client()
+        make_score_sheet_for_one_assignment(creds, gradescope_client = gradescope_client)
+    else:
+        populate_instructor_dashboard()
+
 main()
```

Comment:
```suggestion
if __name__ == "__main__":
    main()
```
https://realpython.com/if-name-main-python/
See GradeView `dbcron\update_bins.py` for another example

Reply:
This is just a suggestion. After you review this, I can approve

Reply:
Thanks for introducing me to if __name__ == "__main__", Mehul! Committed your suggestion.